pkg/status/combinedstatus-resolver.go

```diff
@@ -468,6 +470,60 @@ func (c *combinedStatusResolver) evaluateWorkStatusesPerBindingReadLocked(bindin
 	return combinedStatusesToQueue
 }
 
+// getCombinedContentMap returns a map of content for the given workstatus.
+func (c *combinedStatusResolver) getCombinedContentMap(workStatus *workStatus,
+	resolution *combinedStatusResolution) map[string]interface{} {
+	content := map[string]interface{}{
+		returnedKey:  workStatus.Content(),
```

Comment:
I would make `returned.status` the way to access the workload object's status from the WEC, so that in the future we can easily add access to returned state in other parts of the object.

```suggestion
		returnedKey:  map[string]any{"status": workStatus.Content()},
```

Reply:
This already is what is implemented. `returnedKey` is `"returned"`, and `workStatus::Content` is the function that maps the information returned from WECs:

```
func (ws *workStatus) Content() map[string]interface{} {
	return map[string]interface{}{
		"status": ws.status,
	}
}
```

Reply:
Good. Thanks.